tests/e2e/default/contacts/contact-details.wdio-spec.js

```diff
@@ -61,6 +66,10 @@ describe('Contact details page', () => {
       await commonElements.waitForPageLoaded();
     });
 
+    after(async () => {
+      await utils.revertSettings(true);
+    });
```

Comment:
Do we need this outer `after` function if we are already resetting the settings in the inner after for `Contact summary error`?

Reply:
yes otherwise it throws an error `A previous test did not call revertSettings` when running error test.